tests/testthat/00_create_test_data.R

```diff
@@ -25,7 +25,7 @@ test_that("test data can be created", {
   saveRDS(wp, "data/test_webprocess_knife_prod.rds")
   saveRDS(query(wp, 'algorithms'), "data/test_webprocess_algorithms_prod.rds")
   
-  gconfig(wps.url = "https://cida-test.er.usgs.gov/gdp/process/WebProcessingService")
+  # gconfig(wps.url = "https://cida-test.er.usgs.gov/gdp/process/WebProcessingService")
```

Comment:
oh wow, do we no longer have a test tier for GDP?

Reply:
We do but it's going away and isn't configured right.